assemblies/assembly_decision-manager/main.adoc

```diff
@@ -22,106 +22,22 @@ This document focuses on how to create and test a new project in {CENTRAL} for a
-|fineAmount |Fine Amount |Double
-|totalPoints |Total Points |Integer
-|reason |Reason |String
-
-|===
-
-include::{enterprise-dir}/getting-started/data-object-violation-create-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/data-object-violation-fields-proc.adoc[leveloffset=+3]
-
-include::{enterprise-dir}/getting-started/data-object-driver-create-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/data-object-driver-fields-proc.adoc[leveloffset=+3]
-
-include::{enterprise-dir}/getting-started/guided-traffic-rules-con.adoc[leveloffset=+1]
-
-include::{enterprise-dir}/getting-started/business-rules-create-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/business-rules-conditions-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/business-rules-actions-proc.adoc[leveloffset=+2]
-
-include::{drools-dir}/AuthoringAssets/guided-decision-tables-con.adoc[leveloffset=+1]
-
-include::{enterprise-dir}/getting-started/decision-table-create-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/decision-table-violation-columns-proc.adoc[leveloffset=+3]
-
-include::{enterprise-dir}/getting-started/decision-table-penalty-columns-proc.adoc[leveloffset=+3]
-
-include::{enterprise-dir}/getting-started/decision-table-rows-proc.adoc[leveloffset=+3]
-
-include::{enterprise-dir}/getting-started/test-speed-scenarios-con.adoc[leveloffset=+1]
-
-include::{enterprise-dir}/getting-started/test-speed-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/test-suspend-proc.adoc[leveloffset=+2]
-
-include::{enterprise-dir}/getting-started/test-violation-based-proc.adoc[leveloffset=+2]
-
-// Example projects and decisions:
 include::{drools-dir}/Examples/decision-examples-central-con.adoc[leveloffset=+1]
-
 include::{drools-dir}/Examples/decision-examples-central-proc.adoc[leveloffset=+2]
 
-include::{enterprise-dir}/admin-and-config/assets-executing-proc.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-IDE-con.adoc[leveloffset=+1]
-
-include::{drools-dir}/Examples/decision-examples-IDE-proc.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-helloworld-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-state-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-fibonacci-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-pricing-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-petstore-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-politician-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-sudoku-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-conway-ref.adoc[leveloffset=+2]
-
-include::{drools-dir}/Examples/decision-examples-doom-ref.adoc[leveloffset=+2]
-
 // Versioning info
 include::_artifacts/versioning-information.adoc[]
```

Comment:
I recommend an additional resources section with these items, especially since this is a starting point for users:
```
== Additional resources
* {URL_DMN_MODELS}[_{DMN_MODELS}_]
* {URL_TESTING_DECISION_SERVICE}[_{TESTING_DECISION_SERVICE}_]
* {URL_MANAGING_PROJECTS}[_{MANAGING_PROJECTS}_]
* {URL_KIE_APIS}[_{KIE_APIS}_]
```

Reply:
Changed as suggested.